testData/preferences/snapset_2a.json5

```diff
@@ -25,6 +25,9 @@
                         "http://registry.gpii.net/common/highContrast/enabled": true,
                         "http://registry.gpii.net/common/highContrastTheme": "white-black",
                         "http://registry.gpii.net/common/supportTool": ["dictionary"]
+                    },
+                    "http://registry.gpii.net/applications/net.gpii.psp": {
+                        "http://registry.gpii.net/common/language": "en"
```

Comment:
I think that, according to https://github.com/GPII/universal/pull/640, this should be "en-US", right @danailbd ?

Reply:
Yep, I think it should